File: TestingProjects/ChartGraphView/app/src/main/java/com/example/root/chartgraphview/MainActivity.java
package com.example.root.chartgraphview;

import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.jjoe64.graphview.GraphView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity implements RealtimeUpdatesFragment.OnFragmentInteractionListener {

    Graph graphLogic = new Graph(this);
    GraphView graph;
    public static final String ARG_SECTION_NUMBER = "";

    public void onSectionAttached(int number) {
        ArrayList<String> items = new ArrayList<String>();
        switch (number) {
            case 1:

                break;
            case 2:

                break;
            case 3:

                break;
        }
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        graph = (GraphView) findViewById(R.id.graph);
        graphLogic.onCreate();
    }

    @Override
    protected void onResume() {
        super.onResume();
        graphLogic.onResume();
    }


    @Override
    protected void onPause() {
        super.onPause();
        graphLogic.onPause();
    }


    @Override
    public void onFragmentInteraction(Uri uri) {

    }

}

File: TestingProjects/NetworkFileTest/app/src/main/java/com/ds/da/networkfiletest/MainActivity.kt
package com.ds.da.networkfiletest

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import java.lang.ref.WeakReference
import java.util.*
import android.widget.Toast


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        SendDataButton.setOnClickListener {
            val callAPI = CallAPI()
            callAPI.execute("http://192.168.1.3:8080", "{param: value}")
        }

        retriveDataButton.setOnClickListener {
            val debugTextView = WeakReference(debugTextView)
            RetriveData(debugTextView).execute()
        }

        PostDataButton.setOnClickListener {
            postData().execute()
        }
        uploadFilebutton.setOnClickListener {
            uploadFile().execute()
        }

        timerTaskButton.setOnClickListener {
            val timerObj = Timer()
            val timerTaskObj = object : TimerTask() {
                override fun run() {
                    this@MainActivity.runOnUiThread {
                        Toast.makeText(
                            applicationContext,
                            "timer task running!", Toast.LENGTH_SHORT
                        ).show()
                    }
                }
            }
            timerObj.schedule(timerTaskObj, 0, 5000)
        }
    }
}
